R/initialize_participant_select.R

```diff
@@ -0,0 +1,21 @@
+#' Initialize Participant Select
+#'
+#' @inheritParams shared-params
+#'
+#' @export
+initialize_participant_select <- function(dfAnalyticsInput, session) {
+
+  participant_ids <- dfAnalyticsInput %>%
+    dplyr::filter(.data$GroupLevel == "siteid") %>%
+    dplyr::select("SubjectID") %>%
+    dplyr::arrange(.data$SubjectID) %>%
+    dplyr::pull("SubjectID")
```

Comment:
```suggestion
  participant_ids <- sort(dfAnalyticsInput$SubjectId[dfAnalyticsInput$GroupLevel == "siteid"])
```
To discuss: I feel like base can be easier to deal with when it's something fairly straightforwad like this. Should we have a guideline for when to use base vs tidyverse? It isn't a HUGE deal here (particularly since {gsm} uses {dplyr}, so we can't really eliminate the dependency), just something to work out.

Reply:
Applied (for discussion) in a new `principles.md` file.